scripts/setup.ps1

```
@@ -0,0 +1,205 @@
+    Write-Warn 'rg still is not executable after setup'
+    $Missing.Add('ripgrep (rg)')
+  }
+}
+
+function Ensure-Doppler {
+  Write-Host ''
+  Write-Host '-- Doppler CLI ---------------------------------------------------------'
+
+  if (Test-Command doppler) {
+    Write-Ok "Doppler CLI $(& doppler --version | Select-Object -First 1)"
+    if (& doppler run --project jovie-web --config dev -- cmd /c echo doppler-ok 2>$null) {
```

PSScriptAnalyzer (1.25.0)

[warning] Did you mean to use the redirection operator '>'? The comparison operators in PowerShell are '-gt' (greater than) or '-ge' (greater or equal).

(PSPossibleIncorrectUsageOfRedirectionOperator)